src/core.cpp

```diff
@@ -26,7 +26,7 @@ using namespace goliath;
  * @fn main(int argc, char *argv[])
  * @brief Application entry point
  */
-int main(int argc, char* argv[]) {
+int main(int argc, char *argv[]) {
```

Comment:
What if someone does `sizeof` how would they know it's a pointer and not an array?